custom/src/main/java/com/grafana/extensions/servertiming/ServerTimingHeaderReader.java

```diff
@@ -51,7 +51,7 @@ public <REQUEST, RESPONSE> void consume(
                         GETTER);
             SpanContext spanContext = Span.fromContext(traceparent).getSpanContext();
             if (spanContext.getTraceFlags().isSampled()) {
-              DynamicSampler.setSampled(spanContext.getTraceId());
+              DynamicSampler.getInstance().setSampled(spanContext.getTraceId());
```

Comment:
@zeitlinger can we get a `ConfigProperties` here?

Reply:
you get the properties here: https://github.com/grafana/grafana-opentelemetry-java/blob/03f29b03d742361cae8f3bfdc4e48a67e76c43b3/custom/src/main/java/com/grafana/extensions/sampler/SamplingSpanProcessor.java#L37

then you can store it in the singleton